src/core/layout/scrollable-list/scrollable-list-handlers.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @module core/layout/scrollable-list
+ * @copyright BBC 2020
+ * @author BBC Children's D+E
+ * @license Apache-2.0 Apache-2.0
+ */
+/* eslint-disable no-console */
+
+import fp from "../../../../lib/lodash/fp/fp.js";
+
+const GRID_NAME = "grid";
+
+const handleIfVisible = (gelButton, scene) => {
```

Comment:
Bit unsure what this does exactly. Is it only called on a settings change? It might also be worth it returning a function rather than wrapping it in an anonymous fn later.

Reply:
It checks on click that the button is in visible bounds, i.e. did we click on a button outside of the list's viewable area. This part of the code is incomplete as it just calls a placeholder click handler, but you're right, returning a function would be better. Will refactor.

Reply:
a163e18